cmd/branchingconfigmanagers/tide-config-manager/main.go

```diff
@@ -94,29 +93,40 @@ const (
 	ocpProductName     = "ocp"
 )
 
-func newFeatureFreezeEvent(current, future string) featureFreezeEvent {
-	return featureFreezeEvent{
-		excludedLabels:                sets.NewString(qeApproved, docsApproved, pxApproved),
-		excludedOrgs:                  []string{openshiftPriv},
-		repos:                         sets.NewString(),
-		mainMaster:                    sets.NewString(mainBranch, masterBranch),
-		openshiftReleaseBranches:      sets.NewString(release+current, openshift+current),
-		openshiftReleaseBranchesPlus1: sets.NewString(release+future, openshift+future),
+type sharedDataDelegate struct {
+	excludedLabels sets.String
+	mainMaster     sets.String
+	validBug       sets.String
+}
+
+func newSharedDataDelegate() *sharedDataDelegate {
```

Comment:
This func should be exported otherwise there is no reason to have it at all.

Reply:
I don't think it needs to be exported. It's just a helper to gain more clarity. Common pattern, used eg in [ghproxy](https://github.com/kubernetes/test-infra/blob/4308c4498666d8ec7e51d6b29050d1ec2ef00d73/ghproxy/ghcache/ghcache.go#L178 )